bundles/org.openhab.binding.haassohnpelletoven/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,128 @@
+			<channel id="channelCleaningIn" typeId="cleaningIn"/>
+			<channel id="channelConsumption" typeId="consumption"/>
+			<channel id="channelOnTime" typeId="onTime"/>			
+		</channels>
+
+		<config-description>
+			<parameter name="hostIP" type="text" required="true">
+				<label>IP Address</label>
+				<description>Please add the IP Address of the WIFI Module of the Haas and Sohn oven here</description>
+			</parameter>
+			<parameter name="hostPIN" type="text" required="true">
+				<label>PIN</label>
+				<description>Please add the PIN of your oven here. You can find it in the Menu directly in your oven.</description>
+			</parameter>
+			<parameter name="refreshRate" type="integer">
+				<label>Refresh Rate</label>
+				<description>
+					How often the Pellet Stove should schedule a refresh after a channel is linked to an item. Temperature
+					data will be
+					refreshed according this set time in seconds. Valid input is 0 - 999.
+				</description>
+				<advanced>true</advanced>
+				<default>30</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+	<channel-type id="isTemp">
+		<item-type>String</item-type>
+		<label>Is temperature of the Stove</label>
+		<description>Receives the is temperature as string</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Actual mode of the stove</label>
+		<description>Receives the actual mode of the stove as string</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="spTemp" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Set temperature for the stove</label>
+		<description>Set the target temperature of the stove as number:temperature</description>
+	</channel-type>
+
+	<channel-type id="prg">
+		<item-type>Switch</item-type>
+		<label>On/Off for the stove</label>
+		<description>To turn the stove on/off as switch</description>
+		<event>
+			<options>
+				<option value="ON">ON</option>
+				<option value="OFF">OFF</option>
+			</options>
+		</event>
+	</channel-type>
+	
+	<channel-type id="ecoMode">
+        <item-type>Switch</item-type>
+        <label>On/Off for the Eco Mode of the stove</label>
+        <description>To turn the stove on/off as switch</description>
+        <event>
+            <options>
+                <option value="ON">ON</option>
+                <option value="OFF">OFF</option>
+            </options>
+        </event>
+    </channel-type>
+    
+    <channel-type id="ignitions">
+        <item-type>String</item-type>
+        <label>Amount of ignition's of the stove</label>
+        <description>Receives the total amount of ignition's the stove has as string</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="maintenanceIn">
+        <item-type>String</item-type>
+        <label>States the next maintenance in kg</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
ok